opennmt/utils/decoding.py

```diff
@@ -225,18 +247,22 @@ def finalize(self, outputs, end_id, attention=None, **kwargs):
 class BeamSearch(DecodingStrategy):
     """A beam search strategy."""
 
-    def __init__(self, beam_size, length_penalty=0, coverage_penalty=0):
+    def __init__(
+        self, beam_size, length_penalty=0, coverage_penalty=0, tflite_output=None
```

Comment:
Rename the argument to `tflite_output_size` to be more explicit.

Reply:
I renamed the argument to `tflite_output_size` in the latest commit I just made.